test/compute/jenkins-main-node.test.ts

```diff
@@ -23,7 +23,7 @@ describe('JenkinsMainNode Config Elements', () => {
 
   // THEN
   test('Config elements expected counts', async () => {
-    expect(configElements.filter((e) => e.elementType === 'COMMAND')).toHaveLength(21);
+    expect(configElements.filter((e) => e.elementType === 'COMMAND')).toHaveLength(34);
```

Comment:
Why is this length increased here?

Reply:
Previously we installed the plugins all at once, now I 'spread' out the number of plugins installed at a time to 10 each, adding 13 additional 'install-plugin' cli commands to be run.

Reply:
Got It.